### DISCUSSION TITLE
Showing Disk Label in Key Field

### DISCUSSION THREAD

**Discussion by norsemangrey** [fastfetch-cli/fastfetch#1330] [Q&A] [answered]
Is there any possibility of showing the disk label (e.g. "C:/") in the the `key` field when using a custom key field for disks?

I have tried using something like `"key": "Disk {name:10}"`, but this only gives me the name and not the label.

   Disk   : [ ■■■■■■---- ] 280,76 GiB / 474,66 GiB - NTFS
   Disk   : [ ■■■------- ] 64,56 GiB / 247,88 GiB - ReFS
   Disk   : [ ■--------- ] 78,22 GiB / 683,59 GiB - NTFS

**Comment by CarterLi** [comment, marked as answer]
It is called `mountpoint`

**Reply by norsemangrey** [reply]
Many thanks :)
